docker-compose.template

```diff
@@ -15,6 +22,7 @@ hydroshare:
     POSTGIS_DB: postgres
     POSTGIS_USER: postgres
     PGPASSWORD: postgres
+    SOLR_HOST: hydroshare_solr_1
```

Comment:
Is there a reason this is set as an environment variable, and not added as a link like the other container relationships? Links are automatically added as a host reference that can then be addressed directly using the name defined in the docker-compose file, by docker name, or by container ID.

Example from the hydroshare container running on dev:
```bash
# cat /etc/hosts
172.17.2.103	b5f57cfca1d3
127.0.0.1	localhost
::1	localhost ip6-localhost ip6-loopback
fe00::0	ip6-localnet
ff00::0	ip6-mcastprefix
ff02::1	ip6-allnodes
ff02::2	ip6-allrouters
172.17.2.100	hydroshare_postgis_1 916852b72a87
172.17.2.101	hydroshare_redis_1 2a1734e11821
172.17.2.100	postgis_1 916852b72a87 hydroshare_postgis_1
172.17.2.102	rabbitmq_1 afee239f7baa hydroshare_rabbitmq_1
172.17.2.101	redis 2a1734e11821 hydroshare_redis_1
172.17.2.101	redis_1 2a1734e11821 hydroshare_redis_1
172.17.2.102	hydroshare_rabbitmq_1 afee239f7baa
172.17.2.100	postgis 916852b72a87 hydroshare_postgis_1
172.17.2.102	rabbitmq afee239f7baa hydroshare_rabbitmq_1
```